core/config.class.inc.php

```diff
@@ -1619,6 +1619,14 @@ class Config
 			'source_of_value' => '',
 			'show_in_conf_sample' => false,
 		],
+		'notifications.last_executions_days' => [
+			'type' => 'integer',
+			'description' => 'Number of days to display in the Action\'s last executions tab',
```

Comment:
```suggestion
			'description' => 'Number of days to display in the Action\'s last executions tab (0 means no limit)',
```

Maybe we could precise in the description the behavior of `0`?